src/badguy/willowisp.cpp

```diff
@@ -122,55 +189,66 @@ WillOWisp::active_update(float dt_sec)
   Vector p2 = player->get_bbox().get_middle();
   Vector dist = (p2 - p1);
 
-  switch (m_mystate) {
+  switch (m_mystate)
+  {
     case STATE_STOPPED:
       break;
-
     case STATE_IDLE:
-      if (glm::length(dist) <= m_track_range) {
+      if (glm::length(dist) <= m_track_range)
+      {
         m_mystate = STATE_TRACKING;
       }
       break;
-
     case STATE_TRACKING:
-      if (glm::length(dist) > m_vanish_range) {
+      if (glm::length(dist) > m_vanish_range)
+      {
         vanish();
-      } else if (glm::length(dist) >= 1) {
+      } else if (glm::length(dist) >= 1)
```

Comment:
Sorry, also check the brace placement in this file and all the other ones. Thanks! :)